html/js/melpa.js

```diff
@@ -150,7 +151,7 @@
       pkgs.push(new melpa.Package({
         name: name,
         version: version,
-        commit: built.props.commit,
+        commit: props.commit || '',
```

Comment:
@purcell That's okay. Empty string is [false](https://www.ecma-international.org/ecma-262/7.0/index.html#sec-toboolean).